core/bweb.class.php

```diff
@@ -176,7 +176,7 @@ public function GetVolumeList()
                 // If volume have alreday been used
                 if ($volume['lastwritten'] != "0000-00-00 00:00:00") {
                     // Calculate expiration date only if the volume is Full
-                    if ($volume['volstatus'] == 'Full') {
+                    if ($volume['volstatus'] == 'Full' || $volume['volstatus'] == 'Used') {
```

Comment:
Because the period to expire the volumes just start when the status is Full or Used